manager/src/static/js/marks.js

```diff
@@ -119,6 +125,7 @@ function plot(objects, scale, scene_y_max, svgCanvas, show_telemetry) {
         svgCanvas,
         scale,
         show_telemetry,
+        show_trails
```

Comment:
Please keep the coding convention used in js codebase so far.

The rationale behind:
- Consistency: JS codebase is already using this convention, let's keep it consistent. Also, many style guides and tools (Prettier, ESLint) recommend trailing commas for multi-line objects and arrays.
- Cleaner diffs:
When adding or removing items, only the changed line is modified in version control, making diffs easier to read.
- Easier editing:
You can add new elements without worrying about whether to add or remove a comma from the previous line.


```suggestion
        show_trails,
```